core/src/entities/packages/xrefs.rs

```diff
@@ -0,0 +1,54 @@
+use crate::entities::xrefs::{Xref, XrefKind};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+/// Set of ids used to cross-reference an entity to a [Codebase].
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CodebaseRef {
+    /// Id of the Team that contains the [Project] that contains the [Codebase].
+    pub team_id: String,
+    /// Id of the [Project] that contains the [Codebase]
+    pub project_id: String,
+    /// Id of the [Codebase].
+    pub codebase_id: String,
+}
+
+impl From<CodebaseRef> for Xref {
```

Comment:
@qtpeters  @sbolel  Where did we end up with the Team model?  Is it 2 levels now instead of 3?  What's the child of a Team if so?  Not sure where you are all at in your conversations, but would love to catch up.